### ISSUE TITLE
VM image_locked when Ready VM returns

### ISSUE THREAD

**Issue by aalba6675** [myoung34/vagrant-ovirt3#56]
About 1 in 3 times, Ready VM returns when the VM status is image_locked: this causes failure at VM start.
It is reproducible, just need to vagrant up several times in a row. Somehow Ready VM returns too quickly( the oVirt UI still shows the hour glass).

In start_vm.rb I tried something like this, if you think this is sensible I will send a commit:

```ruby
          # hacky polling; why?
          # I sometimes get image_locked even when Vagrant thinks the VM is ready
          # Start VM.
          begin
            status = "image_locked"
            for i in 1..30
              status = env[:ovirt_compute].servers.get(env[:machine].id.to_s).status
              @logger.info("VM status is #{status}")
              break if status == "down" or status == "up"
              sleep 2
            end

            if status == "down"
              machine.start
            end
          rescue OVIRT::OvirtException => e
            raise Errors::StartVMError,
              :error_message => e.message
          end
```

**Comment by aalba6675** [comment]
I had observed this with the original vagrant-ovirt provider as well.

**Comment by aalba6675** [comment]
I did a several Vagrant up(s) and logged whether the image was locked during Start VM, here is a typical example where the machine status is image_locked
```
$ vagrant up --provider=ovirt3 
Bringing machine 'default' up with 'ovirt3' provider...
==> default: Creating VM with the following settings...
==> default:  -- Name:              example1_146289
==> default:  -- Cpus:              1
==> default:  -- Memory:            1024M
==> default:  -- Template:          Vagrant-CentOS-7
==> default:  -- Version:           base version (1)
==> default:  -- Datacenter:        DC0
==> default:  -- Cluster:           CLUSTER0
==> default:  -- Console:           spice
==> default: Waiting for VM to become "ready" to start...
==> default: Starting VM.
DEBUG: machine status image_locked
DEBUG: machine status image_locked
==> default: Waiting for VM to get an IP address...
==> default: Waiting for SSH to become available...
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine is booted and ready for use!
==> default: Rsyncing folder: /home/vagrant/Projects/Vagrant/example1/vagrant/ => /vagrant
==> default: Configuring and enabling network interfaces...

```

**Comment by myoung34** [comment]
Can you try the `bugfix/56` branch?

**Comment by aalba6675** [comment]
It's working for me. Thanks.

**Comment by myoung34** [comment]
Fixed in #59 . Tagged/pushed as version `1.9.3`
